src/components/primitives/index.ts

```diff
@@ -1,3 +1,19 @@
+// Accordion
```

Comment:
This and `// Drawer` below can be removed (the exports in code indicate what components they are connected to)